src/inference/src/os/win/win_system_conf.cpp

```diff
@@ -58,4 +58,14 @@ int getNumberOfCPUCores(bool bigCoresOnly) {
 std::vector<int> getAvailableNUMANodes() { return {-1}; }
 #endif
 
+bool cpuMapAvailable() {
+    return cpu._cpu_mapping_table.size() > 0;
```

Comment:
Please return false when getAvailableCPUs() for Windows is not ready.

Reply:
Ok, done.